context/context_1.8.2.json

```diff
@@ -0,0 +1,442 @@
+		},
+		"dfc-b:lists": {
+			"@type": "@id"
+		},
+		"dfc-b:localizedBy": {
+			"@type": "@id"
+		},
+		"dfc-b:localizes": {
+			"@type": "@id"
+		},
+		"dfc-b:mainContactOf": {
+			"@type": "@id"
+		},
+		"dfc-b:maintainedBy": {
+			"@type": "@id"
+		},
+		"dfc-b:maintains": {
+			"@type": "@id"
+		},
+		"dfc-b:managedBy": {
+			"@type": "@id"
+		},
+		"dfc-b:manages": {
+			"@type": "@id"
+		},
+		"dfc-b:natureOriginOf": {
+			"@type": "@id"
+		},
+		"dfc-b:nutrientCharacteristicOf": {
+			"@type": "@id"
+		},
+		"dfc-b:objectOf": {
+			"@type": "@id"
+		},
+		"dfc-b:offeredThrough": {
+			"@type": "@id"
+		},
+		"dfc-b:offers": {
+			"@type": "@id"
+		},
+		"dfc-b:offersTo": {
+			"@type": "@id"
+		},
+		"dfc-b:optionOf": {
+			"@type": "@id"
+		},
+		"dfc-b:orderedBy": {
+			"@type": "@id"
+		},
+		"dfc-b:orders": {
+			"@type": "@id"
+		},
+		"dfc-b:outputOf": {
+			"@type": "@id"
+		},
+		"dfc-b:ownedBy": {
+			"@type": "@id"
+		},
+		"dfc-b:owns": {
+			"@type": "@id"
+		},
+		"dfc-b:paidWith": {
+			"@type": "@id"
+		},
+		"dfc-b:partOf": {
+			"@type": "@id"
+		},
+		"dfc-b:partOriginOf": {
+			"@type": "@id"
+		},
+		"dfc-b:phoneNumberOf": {
+			"@type": "@id"
+		},
+		"dfc-b:physicalCharacteristicOf": {
+			"@type": "@id"
+		},
+		"dfc-b:pickedUpAt": {
+			"@type": "@id"
+		},
+		"dfc-b:processOf": {
+			"@type": "@id"
+		},
+		"dfc-b:producedBy": {
+			"@type": "@id"
+		},
+		"dfc-b:produces": {
+			"@type": "@id"
+		},
+		"dfc-b:proposedBy": {
+			"@type": "@id"
+		},
+		"dfc-b:proposes": {
+			"@type": "@id"
+		},
+		"dfc-b:referenceOf": {
+			"@type": "@id"
+		},
+		"dfc-b:referencedBy": {
+			"@type": "@id"
+		},
+		"dfc-b:references": {
+			"@type": "@id"
+		},
+		"dfc-b:refersTo": {
+			"@type": "@id"
+		},
+		"dfc-b:representedBy": {
+			"@type": "@id"
+		},
+		"dfc-b:represents": {
+			"@type": "@id"
+		},
+		"dfc-b:requestedBy": {
+			"@type": "@id"
+		},
+		"dfc-b:requests": {
+			"@type": "@id"
+		},
+		"dfc-b:satisfiedBy": {
+			"@type": "@id"
+		},
+		"dfc-b:satisfies": {
+			"@type": "@id"
+		},
+		"dfc-b:selects": {
+			"@type": "@id"
+		},
+		"dfc-b:socialMediaOf": {
+			"@type": "@id"
+		},
+		"dfc-b:sold": {
+			"@type": "@id"
+		},
+		"dfc-b:soldBy": {
+			"@type": "@id"
+		},
+		"dfc-b:storedIn": {
+			"@type": "@id"
+		},
+		"dfc-b:stores": {
+			"@type": "@id"
+		},
+		"dfc-b:suppliedBy": {
+			"@type": "@id"
+		},
+		"dfc-b:supplies": {
+			"@type": "@id"
+		},
+		"dfc-b:suppliesTo": {
+			"@type": "@id"
+		},
+		"dfc-b:to": {
+			"@type": "@id"
+		},
+		"dfc-b:tracedBy": {
+			"@type": "@id"
+		},
+		"dfc-b:traces": {
+			"@type": "@id"
+		},
+		"dfc-b:transformedBy": {
+			"@type": "@id"
+		},
+		"dfc-b:transforms": {
+			"@type": "@id"
+		},
+		"dfc-b:typeOf": {
+			"@type": "@id"
+		},
+		"dfc-b:uses": {
+			"@type": "@id"
+		},
+
+		"dfc-t:represent": {
+			"@type": "@id"
+		},
+		"dfc-t:hasPivot": {
+			"@type": "@id"
+		},
+		"dfc-t:hostedBy": {
+			"@type": "@id"
+		},
+		"dfc:owner": {
```

Comment:
Maybe include this in the DFC technical ontology (`dfc-t`). This would avoid to forget it if one day we generate the context from the ontology.

Reply:
ok